components/brave_new_tab_ui/components/default/editCards/style.ts

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'brave-ui/theme'
+import palette from 'brave-ui/theme/colors'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  cursor: pointer;
+  margin-top: 5px;
+  border-radius: 8px;
+  backdrop-filter: blur(23px);
+  padding: 12px 20px 8px 20px;
+  background: linear-gradient(90deg, rgba(33, 37, 41, 0.6) 0%, rgba(33, 37, 41, 0.24) 100%);
+
+  &:hover {
+    background: rgba(33, 37, 41, 0.5);
+  }
+
+  &:active {
+    outline-offset: -1px;
+    outline-style: solid;
+    outline-color: ${p => p.theme.color.brandBrave};
+    outline-width: 1px;
+  }
```

Comment:
This is our regular `:focus-visible` state, not `:active`, unless this is what design called for?

Reply:
changed to `:focus`, per design